main/templates/base.html

```diff
@@ -29,10 +29,10 @@
         <a class="nav-link text-light" href="{% url 'workorders_list' %}"><b>Orders</b></a>
       </li>
       <li class="nav-item">
-        <a class="nav-link text-light" href=""><b>Assets</b></a>
+        <a class="nav-link text-light" href="{% url 'assets_list' %}"><b>Assets</b></a>
       </li>
       <li class="nav-item">
-        <a class="nav-link text-light" href=""><b>Parts</b></a>
+        <a class="nav-link text-light" href="{% url 'parts_list' %}"><b>Parts</b></a>
```

Comment:
This will throw an error now that the parts list requires the asset's pk, like mentioned in cmfs.urls.py should use 
`{% url 'parts_all' %}`